apps/fluent-tester/src/FluentTester/TestComponents/Avatar/AvatarTest.tsx

```diff
@@ -1,128 +1,29 @@
 import * as React from 'react';
-import { Avatar, Size } from '@fluentui-react-native/experimental-avatar/';
-import { Text } from '@fluentui/react-native';
-import { Stack } from '@fluentui-react-native/stack';
-import { AVATAR_TESTPAGE } from './consts';
+import { JSAVATAR_TESTPAGE } from './consts';
 import { Test, TestSection, PlatformStatus } from '../Test';
-import { testImageSource, rainbowGradientSource } from './testImageSources';
-import { commonTestStyles as commonStyles } from '../Common/styles';
-import { Switch, View } from 'react-native';
-
-export const BasicAvatar: React.FunctionComponent = () => {
-  const [showImage, setShowImage] = React.useState(true);
-  const [showPresence, setShowPresence] = React.useState(false);
-  const [showRing, setShowRing] = React.useState(false);
-
-  return (
```

Comment:
Can we move the old test to something like "NativeAvatarTest.tsx", have two separate test pages, and then use both on testpages.ios.ts and testpages.macos.ts? I think that would be great for comparing the two avatars

Reply:
I renamed to NativeAvatarTest.tsx. But I don't understand what to do with the separate pages. Should it be similar what I've done to AvatarTest.tsx which imports Basic and Customized Avatar? Or should it be something like NativeAvatarMac.test and NativeAvatarIos.test?

Reply:
On testPages.macos.ts and testpages.ios.ts, could we include both AvatarTest and NativeAvatarTest as two separate test pages?